### DISCUSSION TITLE
cannot read properties of null, reading nextSibling

### DISCUSSION THREAD

**Discussion by jemiton** [nuxt/nuxt#16468] [Questions]
Nuxt 3 app works well on localhost but when I deploy it statically to netlify I get this error screen.  What could be the cause? any solutions?  @manniL 

![image](https://user-images.githubusercontent.com/107219783/200115500-7c8f8e3c-0e7e-455a-b958-79079c2da423.png)

**Comment by TheAlexLichter** [comment]
Hard to tell without reproduction/code. I'd suggest to try locally and check possible hydration errors first.
